mu4/userscores/qml/MuseScore/UserScores/NewScoreDialog.qml

```diff
@@ -42,21 +46,31 @@ QmlDialog {
         Row {
             id: buttons
             anchors.bottom: parent.bottom
-            anchors.bottomMargin: 10
+            anchors.bottomMargin: 16
             anchors.right: parent.right
-            anchors.rightMargin: 10
+            anchors.rightMargin: 16
+
+            spacing: 12
 
-            spacing: 5
+            readonly property int buttonHeight: 30
+            readonly property int buttonWidth: 132
 
             FlatButton {
+                height: buttons.buttonHeight
```

Comment:
It is better to fix the button according to the design dock, and not to edit the width everywhere:)

Reply:
I would prefer not to do it in this PR, because it requires a lot of changes